integreat_compass/cms/models/offers/offer_version.py

```diff
@@ -36,6 +36,7 @@ class OfferVersion(AbstractBaseModel):
         verbose_name=_("Language"),
         help_text=_("The language being taught in this offer"),
     )
+    created_at = models.DateTimeField(default=timezone.now)
```

Comment:
Please correct me if I'm wrong, but to me this field seems pretty identical to [offer_version_date](https://github.com/digitalfabrik/integreat-compass/blob/0843e214ba5f386c3df3fa513e63f6ba15aca647/integreat_compass/cms/models/offers/offer_version.py#L20). If the fields do the same thing, I would suggest to either rename the field from offer_version_date to create_at if that makes more sense to you/us or remove the field again

Reply:
Good point, fixed it.